pages/index.js

```diff
@@ -2,36 +2,55 @@ import HeadObject from "../components/head";
 import Nav from "../components/nav";
 import Scene from "../components/scene";
 import Balancer from "react-wrap-balancer";
+import Link from "next/link";
 
 export default function Home() {
   return (
     <div className="dark:text-white dark:bg-black">
-      <HeadObject />
-      <a href="https://hackclub.com"><img src="https://assets.hackclub.com/flag-orpheus-top.svg" className="absolute top-0 left-8 w-1/4 md:w-[14%]" /></a>
+      {/* <HeadObject />
+      <Link href="https://hackclub.com">
+        <img
+          src="https://assets.hackclub.com/flag-orpheus-top.svg"
+          className="absolute top-0 left-8 w-1/4 md:w-[14%]"
+        />
+      </Link> */}
+
+      <div>
+        <a class="banner mx-auto ml-2 z-50" href="https://hackclub.com/">
+          <img
+            src="https://contribute.hackclub.com/images/flag-orpheus-top.png"
+            alt="Hack Club Logo"
+          ></img>
+        </a>
+      </div>
+
       <main>
-        <section
-          id="hero"
-          className="flex flex-col items-center relative"
-        >
-          {/* <Nav /> */}
+        <section id="hero" className="flex flex-col items-center relative">
           <div className="my-24 flex flex-col gap-4 items-center text-center z-10">
-            <h1 className="text-9xl">
+            <h1 className="text-6xl sm:text-9xl">
               Ham
               <br />
               Club
             </h1>
-            <p className="text-4xl">
+            <p className="text-2xl sm:text-4xl">
```

Comment:
Can you explain why this change is needed?

Reply:
For mobile

Reply:
<img width="327" alt="image" src="https://github.com/hackclub/ham-club/assets/83499056/3289293b-241f-425b-ab0f-2d7c48e745d8">
this isn't ideal

Reply:
what does it look like once you added that? and does it affect the larger version of the site?

Reply:
![image](https://github.com/hackclub/ham-club/assets/83499056/f311c970-c570-4b86-8d3e-7d6aeff922b5)
It does not affect the size on the larger version of the site.